cmd/pr.go

```diff
@@ -99,6 +99,9 @@ func PrCommand(repo *core.Repo, gh func(context.Context) core.Gh) *cli.Command {
 			if core.GetCleanMaster() && !args.Detached {
 				// We need to remove the commits we used to make the PR from the branch
 				// we were on before.
+				if !repo.NoLocalChanges(cCtx.Context) {
```

Comment:
This should be in sanitize args no ? It's the place where we do all the logic of checking the args etc.